web/src/main/resources/applicationContext-web.xml

```diff
@@ -67,5 +67,10 @@
     <mvc:interceptors>
         <bean id="involvedCancerStudyInterceptor" class="org.cbioportal.web.util.InvolvedCancerStudyExtractorInterceptor"/>
     </mvc:interceptors>
+    <beans profile="ga-tracking-enabled">
```

Comment:
ga-api-tracking-enabled?  i think we should be careful to distinguish api tracking from existing frontend ga implementation

Reply:
Understood.  Easy change.